docs/user/openstack/configure_host_device_network.md

```diff
@@ -0,0 +1,9 @@
+# Expose host-device interface to pod
+
+The host-device CNI plugin can be used to exposed an interface on the host to the pod. The plugin moves the 
+interface from the host network namespace to the pods namespace. Effectively given direct control of the interface 
+to the pod running in the namespace. 
+
+[Creating an additional network attachment with the host-device CNI plug-in] [1]
```

Comment:
Was going to note the space between these, but it seems Markdown handles that just fine